zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -115,4 +115,8 @@ angular.module('zeppelinWebApp').controller('NavCtrl', function($scope, $rootSco
   vm.loadNotes();
   $scope.checkUsername();
 
+  $rootScope.dateToString = function(date) {
```

Comment:
This is not the right place for this function, also since its used only once in the code base, you could just use it inline.